src/LinuxTracing/SwitchesStatesNamesVisitorTest.cpp

```diff
@@ -451,12 +481,12 @@ TEST_F(SwitchesStatesNamesVisitorTest, NoThreadStatesOfUninterestingPid) {
 }
 
 TEST_F(SwitchesStatesNamesVisitorTest, VariousThreadStateSlicesFromAllPossibleEvents) {
-  visitor_.SetThreadStatePidFilter(kPid);
+  visitor_.SetThreadStatePidFilters({kPid1, kPid2});
```

Comment:
This is strange indeed, I need to look deeper into it. The events are for different threads, I'm surprised that this would cause an issue. Will resume investigation tomorrow. Thanks for pointing that out.